### PR TITLE
Add formatting to simulation mode helptext

### PR DESCRIPTION
**Pull request by andreas-el**
Make the help text for easy on the eye, and also highlight the limitation imposed on `test-run` simulation mode.

The most prominent changes;

Before;
Screenshot 2024-09-27 at 14 37 41

After:
Screenshot 2024-09-27 at 13 50 10

**Issue**
Resolves #my_issue

**Approach**
_Short description of the approach_

(Screenshot of new behavior in GUI if applicable)

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Make sure unit tests pass locally after every commit (`git rebase -i main
      --exec 'pytest tests/ert/unit_tests -n logical -m "not integration_test"'`)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: src/ert/run_models/single_test_run.py [hunk 1]
```diff
@@ -16,9 +16,9 @@
 class SingleTestRun(EnsembleExperiment):
     """
     Single test is equivalent to EnsembleExperiment, in that it
-    samples the prior and evaluates it. There are two key differences:
-    1) Single test run always runs locally using the local queue
-    2) Only a single realization is run
+    samples the prior and evaluates it.<br>There are two key differences:<br>
+    1) Single test run always runs locally using the <b>local queue</b><br>
+    2) Only a <b>single realization</b> is run<br>
```
Comment (berland) [inline src/ert/run_models/single_test_run.py:21]: Could we also indicate that it is always realization 0 that is being run? (it matters for those running design matrices)

Comment (andreas-el) [reply src/ert/run_models/single_test_run.py:21]: Sure thing!

Comment (andreas-el) [reply src/ert/run_models/single_test_run.py]: Screenshot 2024-09-27 at 15 38 53

If you have a proposal, I'm all ears. 👂

**Review by berland — APPROVED** [review]
yay! 🕺🏻

**Pull request merged**